### ISSUE TITLE
Is there a way to generate these codes on NodeJs, as we cannot use dom there?

### ISSUE THREAD

**Issue by aliarshad9691** [252860883/ArtQRCode#8]
I am trying to generate bulk codes in a project.
generating them on browser does not seems a good idea...
Is there a possibility to generate codes server side?

**Comment by tazim404** [comment]
Yes, it is possible you need to take the data from users and send it back to the server where QRcode will be generated and they will be stored as temporary files just (like we do with bytes io in python) and send back to the client.

**Comment by XingXiaoWu** [comment]
maybe u can use Headless browser

**Comment by aliarshad9691** [comment]
I am able to generate on server side, by adding follwoing lines on top of artqrcodejs

```
import {Canvas} from 'canvas';
import {JSDOM} from 'jsdom';
const dom = new JSDOM('<!DOCTYPE html><div id="qr"></div>');
var document = dom.window.document;
global.Image = Canvas.Image;
```

i'll make a PR, so this library can be used on server side as well. This is very helpful for bulk qrCode generation.
